src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -210,12 +238,30 @@ private static void init(){}
         public static final TagKey<Item> COBBLESTONE_INFESTED = tag("cobblestone/infested");
         public static final TagKey<Item> COBBLESTONE_MOSSY = tag("cobblestone/mossy");
         public static final TagKey<Item> COBBLESTONE_DEEPSLATE = tag("cobblestone/deepslate");
+        public static final TagKey<Item> CLUSTERS = tag("clusters");
```

Comment:
i feel like clusters might be an unintutive enough name that mentioning "like amethyst" would help. Same with "budding blocks" and "buds"